edb/edgeql-parser/src/tokenizer.rs

```diff
@@ -809,6 +809,7 @@ pub fn is_keyword(s: &str) -> bool {
         | "__subject__"
         | "__type__"
         | "__std__"
+        | "abort"
```

Comment:
This seem to be unrelated to the PR

Reply:
I synced the current list of keywords (forgot to in the original migration pr).